os.bpmsuite.smartrouter/added/launch/bpmsuite-smartrouter.sh

```diff
@@ -17,6 +17,8 @@ function prepareEnv() {
     unset KIE_SERVER_ROUTER_PORT
     unset KIE_SERVER_ROUTER_PROTOCOL
     unset KIE_SERVER_ROUTER_URL_EXTERNAL
+    unset KIE_SERVER_ROUTER_REPO_PATH
+    unset KIE_SERVER_ROUTER_CONFIG_WATCH
```

Comment:
KIE_SERVER_ROUTER_CONFIG_WATCHER_ENABLED, per the comment above about KIE_SERVER_ROUTER_REPO.